CSSerialProtocol/CSSerialProtocol.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
It would be helpful to have comments documenting which of these functions should be used by consumers of this library